README.md

```diff
@@ -1 +1,24 @@
-# jenkins-job-dsl
\ No newline at end of file
+# jenkins-job-dsl
+
+## File structure
+
+    .
+    ├── <project>                       # Example project name e.g. mobile app
+    │   ├── jobs                        # Job DSL code
+    │   ├── resources                   # Resources specific to this project
+    ├── <other_project>                 # Another project name e.g. analytics
+    │   ├── jobs                        # Job DSL code
+    │   ├── resources                   # Resources specific to this project
+    ├── <...>                           # A folder for every project!
+    ├── shared                          # Code shared between each project
+    │   ├── main
+    │   │   ├── groovy                  # support classes
+    │   │   └── resources
+    │   │       └── idea.gdsl           # IDE support for IDEA
+    │   └── test
+    │       └── groovy                  # specs
+    └── build.gradle                    # build file
+    └── hosts
```

Comment:
I could also see this as `seeds` which could arguably be run anywhere and don't tie us to specific URLs.